mantidproject/mantid
# Reflectometry GUI restructuring 8: Postprocess groups

**Description of work.**
Ongoing work to integrate the `JobTreeView` into the `ISIS Reflectometry` GUI. This PR is being merged to a feature branch because not all functionality is implemented yet.

This PR implements postprocessing (stitching) of row outputs in the `ISIS Reflectometry` GUI.

**To test:**

- Open the `ISIS Reflectometry` interface
- Set the instrument to `INTER` and enter the following information in the table:
![image](https://user-images.githubusercontent.com/12895056/53328801-af84ef80-38e2-11e9-9974-79b7257ad3ae.png)
- Click `Process` and the rows should be reduced and turn green.
- The Group should be postprocessed and also turn green.
- You should have the following output workspaces:
![image](https://user-images.githubusercontent.com/12895056/53328915-e6f39c00-38e2-11e9-92d8-ee4f6e26552a.png)

Fixes #24804 

*This does not require release notes* because it only involves under-the-hood changes

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
